packages/puppeteer-core/src/cdp/Page.ts

```diff
@@ -352,7 +352,6 @@ export class CdpPage extends Page {
       this.#workers.set(session.id(), worker);
       this.emit(PageEvent.WorkerCreated, worker);
     }
-    session.on(CDPSessionEvent.Ready, this.#onAttachedToTarget);
```

Comment:
I believe it's a subframe session here that might need to be configured recursively

Reply:
ahhh because the session will notify that another session is ready. Gotcha!